core_backend/app/schemas.py

```diff
@@ -111,7 +111,7 @@ class ContentCreate(BaseModel):
     # Ensure len("*{title}*\n\n{text}") <= 1600
     content_title: Annotated[str, StringConstraints(max_length=150)]
     content_text: Annotated[str, StringConstraints(max_length=1446)]
-    content_language: str
+    content_language: str = "ENGLISH"
```

Comment:
I added a default value until we update the admin app. So that the admin app don't  crash.